include/bluetooth/bluetooth.h

```diff
@@ -664,18 +670,55 @@ struct bt_le_scan_param {
 
 	/** Scan window (N * 0.625 ms) */
 	u16_t window;
+
+	/** Scan timeout (N * 10 ms)
+	 *  Application will be notified by the scan timeout callback.
+	 *  Set zero to disable timeout.
+	 */
+	u16_t timeout;
+
+	/** Scan interval coded (long range) (N * 0.625 MS)
+	 *  Set zero to use same as 1M scanning
+	 */
+	u16_t interval_coded;
+
+	/** Scan window coded (long range) (N * 0.625 MS)
+	 *  Set zero to use same as 1M scanning
+	 */
+	u16_t window_coded;
 };
 
 /** LE advertisement packet information */
 struct bt_le_adv_info {
-	/** Advertiser LE address and type */
+	/** Advertiser LE address and type. */
 	const bt_addr_le_t *addr;
 
-	/** Strength of advertiser signal */
+	/** Directed address. */
+	const bt_addr_le_t *direct_addr;
+
+	/* Advertising Set Identifier. */
+	u8_t sid;
+
+	/** Strength of advertiser signal. */
 	s8_t rssi;
 
-	/** Advertising packet type */
+	/** Transmit power of the advertiser. */
+	s8_t tx_power;
+
+	/** Advertising packet type. */
 	u8_t adv_type;
+
+	/** Advertising packet properties. */
+	u16_t adv_props;
+
+	/* Primary advertising channel PHY. */
+	u8_t primary_phy;
+
+	/* Secondary advertising channel PHY. */
+	u8_t secondary_phy;
+
+	/* Periodic advertising interval of Periodic Advertiser. */
+	u16_t per_interval;
```

Comment:
remove `per_interval`. Let's add those later once we implement periodic advertising